src/frontend/context/StegContext.tsx

```diff
@@ -10,7 +9,23 @@ import { useApp } from './AppContext';
 import { useEøs } from './EøsContext';
 import { useRoutes } from './RoutesContext';
 
-const [StegProvider, useSteg] = createUseContext(() => {
+interface StegContext {
+    steg: ISteg[];
+    barnForSteg: IBarnMedISøknad[];
+    settBarnForSteg: React.Dispatch<React.SetStateAction<IBarnMedISøknad[]>>;
```

Comment:
Ingen stor sak, men tror dette vil fungere om du ikke vil ha tilknytningen til `React.Dispatch` og `React.SetStateAction` 

```suggestion
    settBarnForSteg: (barnMedSøknad: IBarnMedISøknad[]) => void;
```

Reply:
Enig! Jeg fikser! 🙌